Todo.py

```diff
@@ -0,0 +1,203 @@
+            newEmbed.add_field(name="Public repost", value=repostChannel.mention + " : " + str(publicTodo.id), inline=True)
+        
+        message = await channel.send(embed=newEmbed)
+        await message.add_reaction(EMOJIS['wastebasket'])
+        await message.add_reaction(EMOJIS['check'])
+        await message.add_reaction(EMOJIS['hourglass'])
+        await ctx.message.delete()
+
+    @todo.command()
+    async def addtype(self, ctx, command):
+        '''Command to add a todo type.'''
+        command = command.split(";")
+
+        if command[1].startswith("#"):
+            command[1] = command[1][1:]
+        if len(command[1]) != 6:
+            if len(command[1]) != 3:
+                await ctx.send('The color must be in hexadecimal, like this **#ccc** or **#ff0000**')
+                return
+            else:
+                command[1] = command[1]+command[1]
+        color = "0x" + command[1]
+
+        with open(TODO_TYPES, "r+") as file:
+            content = file.readlines()
+            for line in content:
+                line = line.split(";")
+                if line[0] == command[0]:
+                    await ctx.send('There is already a type named **'+command[0]+'**')
+                    return
+            
+            file.write('\n' + command[0] + ';' + color)
+        await ctx.send('You added the label "'+command[0]+'", the embed\'s color for this todo type will be : #' + command[1])
+
+    @todo.command()
+    async def removetype(self, ctx, command):
+        '''Command to add a remove type.'''
+        with open(TODO_TYPES, "r") as file:
+            lines = file.readlines()
+        with open(TODO_TYPES, "w") as file:
+            deleted=False
+            for line in lines:
+                line = line.split(";")
+                if line[0] != command:
+                    file.write(';'.join(line))
+                else:
+                    deleted=True
+        
+        if deleted:
+            await ctx.send('Todo type **'+command+'** deleted')
+        else:
+            await ctx.send('There is no type named **'+command+'**')
+
+    @todo.command()
+    async def listtypes(self, ctx):
+        '''Command to list all the todo types.'''
+        text = "**Type** - *Color*\n\n"
+        with open(TODO_TYPES, "r") as file:
+            lines = file.readlines()
+            for line in lines:
+                line = line.split(';')
+                text += "**" + line[0] + "** - *#"+line[1][2:] + "*\n"
+
+        newEmbed = discord.Embed(description=text, url="", color=0x28a745)
+        message = await ctx.channel.send(embed=newEmbed)
+
+    @todo.command()
+    async def channel(self, ctx):
+        '''Command to select the channel where the todos will be'''
+        with open(TODO_CHANNEL, "w") as file:
+            file.write(str(ctx.channel.id))
+        await ctx.channel.send('Okay ! This channel wil be used for the todos !')
+
+def setup(bot):
+    bot.add_cog(Todo(bot))
```

Comment:
It is recommended to add a newline at the end of a file.
```suggestion
    bot.add_cog(Todo(bot))
    
```

Reply:
Same as your last change :D